Tests/XMTPTests/ConversationsTest.swift

```diff
@@ -193,4 +193,21 @@ class ConversationsTests: XCTestCase {
 			}
 		}
 	}
+	
+	func testLoadAllConvosForLargeWallet() async throws {
+		// Data from hex: 0836200ffafa17a3cb8b54f22d6afa60b13da48726543241adc5c250dbb0e0cd
+		// aka 2k many convo test wallet
+		let privateKeyData = Data([8,54,32,15,250,250,23,163,203,139,84,242,45,106,250,96,177,61,164,135,38,84,50,65,173,197,194,80,219,176,224,205])
```

Comment:
One worry about including this PK in the repo is that random devs are going to use it and start adding more convos to the account, or even misusing it to accidentally have broken convos. That makes it less useful as a testing tool.

Reply:
Ya I've been hesitant to commit any of these for awhile. But they keep creeping up with issues. I'll remove it for now. A good test that creates 101 convos may suffice. Atleast for pagination if not for performance.

Reply:
➕ on a 101 conversation test